src/lib/deal.ts

```diff
@@ -90,11 +152,24 @@ export const dealUpdate = async ({
   appCID,
 }: DealUpdateArg) => {
   const signer = await getSigner(network, privKey);
-  const deal = getDealContract(dealAddress, signer);
+
+  const dealClient = new DealClient(signer, network);
+  const deal = dealClient.getDeal(dealAddress);
+
+  const config = await deal.getConfigModule();
+
+  const bytesCid = CID.parse(appCID).bytes;
 
   promptConfirmTx(privKey);
 
-  const tx = await deal.setAppCID(appCID);
+  commandObj.log(`${bytesCid.length}`);
```

Comment:
is this required? if you want this to still exist maybe it should be more descriptive and use logToStderr